### DISCUSSION TITLE
Php 8 support

### DISCUSSION THREAD

**Discussion by carlittoer** [smarty-php/smarty#676] [Q&A] [answered]
Hi,

When is Smarty(4) with php 8 support coming out?

When i run the latest smarty with php 8, it throws lots of warnings.

**Comment by wisskid** [comment, marked as answer]
Hi, check out the pre-release and https://github.com/smarty-php/smarty/discussions/679

**Reply by wisskid** [reply]
Smarty v4 is now official, supporting PHP8.
